src/init.cpp

```diff
@@ -1713,8 +1699,14 @@ bool AppInitMain(InitInterfaces& interfaces)
 #endif
 
     std::vector<fs::path> vImportFiles;
-    for (const std::string& strFile : gArgs.GetArgs("-loadblock")) {
-        vImportFiles.push_back(strFile);
+
+    if (gArgs.IsArgSet("-loadblock")) {
+        const std::vector<std::string> vLoadBlockArgs = gArgs.GetArgs("-loadblock");
+
+        // Regard the '-noloadblock' argument
```

Comment:
what does this comment mean? `GetArgs` takes care of the `-noloadblock` option

Reply:
Also, why is this code block changed at all?

Reply:
True, after implementing the feedback this doesn't do anything different aside from using `AbsPathForConfigVal()` which does not seem to be critical here.